tests/testthat/test-estimate_delay.R

```diff
@@ -182,38 +182,7 @@ test_that("estimate_delay handles diagonal reporting triangles", {
   )
 })
 
```

Comment:
Tests were removed here but not added to `baselinenowcast.reporting_triangle()`. 
It would be good to test that in those functions when passing in a reporting triangle with negatives `baselinenowcast.reporting_triangle(preprocess = NULL)` preserves negative values, and with the default (`preprocess = `preprocess_negative_values()`) the function returns something with all positive entries. 

Also could add in there the test that was removed here for custom preprocessing?